Warning in AssetElementValueConverter

### Brief bug description

What went wrong?

`AssetElementValueConverter.GetPropertyValueAsync` is marked as async but does not use await.

### Repro steps

1. Clone the repository
2. Build the solution
see

```plain
C:\projects\kontent-delivery-sdk-net\Kentico.Kontent.Delivery\ContentItems\AssetElementValueConverter.cs(21,35): warning CS1998: This async method lacks 'await' operators and will run 
synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [C:\projects\kontent-delivery-sdk-net\Kentico.Kontent.Delivery\Kentico.Kontent.Delivery.csproj]
C:\projects\kontent-delivery-sdk-net\Kentico.Kontent.Delivery\ContentItems\AssetElementValueConverter.cs(21,35): warning CS1998: This async method lacks 'await' operators and will run 
synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [C:\projects\kontent-delivery-sdk-net\Kentico.Kontent.Delivery\Kentico.Kontent.Delivery.csproj]
```

### Expected behavior

No warning

### Context

Related to #318

--- a/Kentico.Kontent.Delivery/ContentItems/AssetElementValueConverter.cs
+++ b/Kentico.Kontent.Delivery/ContentItems/AssetElementValueConverter.cs
@@ -18,7 +18,7 @@ namespace Kentico.Kontent.Delivery.ContentItems
             Options = options;
         }
 
-        public async Task<object> GetPropertyValueAsync<TElement>(PropertyInfo property, TElement contentElement, ResolvingContext context) where TElement : IContentElementValue<IEnumerable<IAsset>>
+        public Task<object> GetPropertyValueAsync<TElement>(PropertyInfo property, TElement contentElement, ResolvingContext context) where TElement : IContentElementValue<IEnumerable<IAsset>>
         {
             if (!typeof(IEnumerable<IAsset>).IsAssignableFrom(property.PropertyType))
             {
@@ -30,7 +30,7 @@ namespace Kentico.Kontent.Delivery.ContentItems
                 return null;
             }
 
-            return assetElementValue.Value
+            var assets = assetElementValue.Value
                 .Select(asset => new Asset
                 {
                     Description = asset.Description,
@@ -44,6 +44,8 @@ namespace Kentico.Kontent.Delivery.ContentItems
                 })
                 .Cast<IAsset>()
                 .ToList();
+
+            return Task.FromResult((object)assets);
         }
 
         private string ResolveAssetUrl(IAsset asset)

